### DISCUSSION TITLE
Welcome to predector Discussions!

### DISCUSSION THREAD

**Discussion by darcyabjones** [ccdmb/predector#53] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paulamool** [comment]
Hello, I'm Paula (Curtin University Research Fellow). Currently running Predector on _Pyrenophora_ species and others!

**Reply by darcyabjones** [reply]
Oh hello! I didn't realise it would send notifications out!

Wondering if this will be easier for people to ask quick questions on. A bit like some other projects have Gitters.
Of course that's assuming people actually us this :)

**Comment by paulamool** [comment]
This is great. A good way to give feedback and ask questions rather than issues?

**Reply by darcyabjones** [reply]
Yeah, it's a nice feature.
I think I'd normally prefer issues for bugs and feature requests. but i think this would work for quick points of clarification or general help. A bit like slack?

I guess people might it less of a barrier than creating an issue.

Disappointed that I can't set it to not require a gh login.
I mean I get why but I some people will just email us directly if there's that kind of barrier.

**Reply by fmobegi** [reply]
An email notification brought me here and I must say this is a better engagement feature separate from "Issues" reporting. Anyways, my analyses on _Ascochyta rabiei_ went well
